Readme.md

```diff
@@ -11,6 +11,30 @@
 $ npm install koa-response-time
 ```
 
+## Usage
+
+Basic usage:
+
```

Comment:
pls, add a markdown highlight :).

Reply:
Added. 
PS: Actually four space also hight code in markdown.